docs/04-Patterns/01-duration-terms.adoc

```diff
@@ -4,7 +4,7 @@
 |===
 
 === Begriffe und Konzepte
-Resilienz Muster, Container Application Design, Cloud Native Architekturen, Container Pattern, Service Mesh
+Resilienz Muster, Container Application Design, Cloud Native Architekturen, Functions as a Service (FaaS), Container Pattern, Service Mesh
```

Comment:
Maybe it is better to change the order like Container Pattern, Functions as a Service (FaaS), cause FaaS is based on the Container Pattern